### PR TITLE
refactor: HTTP error codes POST /mentorship_relation/send_request

### LINKED ISSUE #626: Fix: refactor HTTP error codes POST /mentorship_relation/send_request
## Description
As a GSoC student,
I need the correct HTTP error response code to be returned accordingly by Mentorship System API in relation to POST /mentorship_relation/send_request,
so that I can represent the error message accordingly to both Mentorship System and Bridge In Tech users.

## Mocks
NA

### Update [Required]
- [ ] 201: Created need to be returned instead of 200: OK if the process is successful

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by satya7289**
### Description

Refactor HTTP status code from 200->201(ok->created) @ POST /mentorship_relation/send_request

Fixes #626

### Type of Change:

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Screenshot from 2020-06-19 20-31-04

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Update Swagger documentation and the exported file at /docs folder

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/mentorship_relation/test_api_send_request.py [hunk 1]
```diff
@@ -15,6 +15,21 @@ class TestSendRequestApi(MentorshipRelationBaseTestCase):
...
+        test_payload={
+            "mentor_id": self.first_user.id,
+            "mentee_id": self.second_user.id,
+            "end_date": int((datetime.now()+timedelta(days=40)).timestamp()),
+            "notes": "some notes"
+        }
+        actual_response = self.client.post('/mentorship_relation/send_request',
+                                           headers=auth_header, content_type='application/json',
+                                           data=json.dumps(test_payload))
+        self.assertEqual(201, actual_response.status_code)
```
Comment (isabelcosta) [inline tests/mentorship_relation/test_api_send_request.py]: Can you use `HTTPStatus.CREATED` instead of 201?

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@satya7289 i left a little change request, after that is done, I would be more than happy to approve it 💪

**Review by isabelcosta — APPROVED** [review]
Looks good @satya7289 💯 
@anitab-org/coding-team could you please help reviewing this :)

**Review by robotjellyzone — APPROVED** [review]
LGTM, 👍 @satya7289

**Comment by isabelcosta** [comment]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

  * _Sent a request from Catarina to another user_  
    _Screenshot/gif_:  
send request proof of success on Swagger UI
    _Expected Result_:  Get 201 CREATED as HTTP status code
    _Actual Result_: Got 201 as HTTP status code
    
3. Additional testcases covered:

N/A
    
4. Additional Comments: Tested on the deployment environment of this PR https://ms-backend-review-pr-649.herokuapp.com/ 🎉 

5. Status of PR Changed to: Ready to Merge.

**Comment by isabelcosta** [comment]
@robotjellyzone thank you so much for your review!
Thank you for your contribution @satya7289 🙌

**Pull request merged**